src/vanilla/store.ts

```diff
@@ -241,9 +235,12 @@ const flushBatch = (batch: Batch) => {
   }
   while (batch.M.size || batch.L.size) {
```

Comment:
Sorry, I forgot this in previous PR.

```suggestion
  while (batch.H.size || batch.M.size || batch.L.size) {
```